Description: Disable unavailable dep and feature tzdb
Forwarded: not-needed
Author: Blair Noctis <ncts@debian.org>
Last-Update: 2025-03-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -51 +51 @@
-    "tzdb",
+    #"tzdb",
@@ -57 +57 @@
-    "tzdb",
+    #"tzdb",
@@ -112 +112 @@
-[dependencies.tzdb]
+[disabled.dependencies.tzdb]
